.github/workflows/e2e.yml

```diff
@@ -151,11 +156,11 @@ jobs:
             --use-param-defaults \
             --param git_repo_url=git@github.com:redhat-openshift-ecosystem/operator-pipelines-test.git \
             --param upstream_repo_name=redhat-openshift-ecosystem/operator-pipelines-test \
-            --param git_branch=$SHORT_SHA \
-            --param bundle_path=operators/test-e2e-operator/0.0.7-$SHORT_SHA \
+            --param git_branch=$VERSION \
+            --param bundle_path=operators/test-e2e-operator/$VERSION \
             --param env=stage \
             --param submit=false \
-            --param pipeline_image=quay.io/redhat-isv/operator-pipelines-images:${{ github.sha }} \
+            --param pipeline_image=quay.io/redhat-isv/operator-pipelines-images:latest \
```

Comment:
Do you intend to revert this change in all three pipeline start commands?

Reply:
It is for testing purposes only. I will change it before merging the PR.